landing_page_backend/landing_page_backend/settings.py

```diff
@@ -125,7 +133,8 @@
 
 EMAIL_HOST = 'smtp.ukr.net'
 EMAIL_PORT = 465
-EMAIL_HOST_USER = 'testsmtpmail1@ukr.net'
-EMAIL_HOST_PASSWORD = 'WNCr2QnsV3dn8CTL'
+EMAIL_HOST_USER = env('EMAIL_HOST_USER')
```

Comment:
Project will fail with no env variables set up, include .env.example file with empty values to git.